python/hsfs/feature_group.py

```diff
@@ -116,8 +116,35 @@ def select_all(self, include_primary_key=True, include_event_time=True):
 
             # show first 5 rows
             query.show(5)
+
+
+            # select all features exclude primary key and event time
+            from hsfs.feature import Feature
+            fg = fs.create_feature_group(
+                    "fg",
+                    features=[
+                            Feature("id", type="string"),
+                            Feature("ts", type="bigint"),
+                            Feature("f1", type="date"),
+                            Feature("f2", type="double")
+                            ],
+                    primary_key=["id"],
+                    event_time="ts")
+
+            query = fg.select_all()
+            query.features
+            # [Feature('id', ...), Feature('ts', ...), Feature('f1', ...), Feature('f2', ...)]
```

Comment:
This won't work as you have to call `fg.insert` to actually create the FG. I think we can keep it short as in the example above and just assume the feature group exists and do `fs.get_or_create_feature_group(...)`

Reply:
This example works without actually creating a fg. 
I think it is good to show the feature definition, including name, primary key and event time for demonstrating how `include_primary_key` and `include_event_time` work.

Reply:
aha ok but you cannot execute the query